lib/pipedproviders.js

```diff
@@ -36,7 +36,14 @@ module.exports = function (app) {
 
     result.pipeElements[result.pipeElements.length - 1].pipe(new DevNull())
     result.pipeElements[result.pipeElements.length - 1].on('data', msg => {
-      app.handleMessage(providerConfig.id, msg)
+      let provider = providerConfig.id
```

Comment:
For some reason this goes against my grain. If one provider is really several sources shouldn't we have a data structure that mirrors that, instead of overriding the provider id?

Reply:
I knew you weren't going to like that. I'm not sure I understand what you're suggesting.

Reply:
I tried to say that if this is a single Provider then it should show up as a single Provider with a single provider id. If we need to delve deeper into it in the ui we should create some ui elements and the corresponding data structures to reflect that and not just misuse the existing providerId.

No great ideas how exactly we should do it though.